package.json

```diff
@@ -61,6 +73,7 @@
     "brotli": "^1.3.2",
     "codecov": "^3.8.2",
     "conventional-changelog-cli": "^2.1.1",
+    "esbuild": "^0.12.25",
```

Comment:
You were missing this as a dependency. I suspect you have it installed globally on your machine. I think this is a safer, more encapsulated option, especially for inviting contributors.